src/node_metadata.h

```diff
@@ -67,7 +73,8 @@ namespace node {
   NODE_VERSIONS_KEYS_BASE(V)                                                   \
   NODE_VERSIONS_KEY_CRYPTO(V)                                                  \
   NODE_VERSIONS_KEY_INTL(V)                                                    \
-  NODE_VERSIONS_KEY_QUIC(V)
+  NODE_VERSIONS_KEY_QUIC(V)                                                    \
+  NODE_VERSIONS_UVWASI(V)
```

Comment:
For consistency with the lines above it:

```suggestion
  NODE_VERSIONS_KEY_UVWASI(V)
```